app/templates/public/sessions.hbs

```diff
@@ -53,15 +53,19 @@
 
 <div class="mt-8">
   {{#if this.model.session}}
+  {{#each this.groupByDateSessions as |group|}}
+    <h3 class="ui header" style="margin-bottom: -20px; color: rgba(0, 0, 0, 0.7);">{{t-var group.date}}</h3>
     <table class="ui very basic table">
-      <thead class="full-width">
-        <tr>
-          <th class="two wide" style="border: none;"></th>
-          <th style="border: none;"></th>
-        </tr>
-      </thead>
+      {{#unless this.device.isMobile}}
```

Comment:
Why?

Reply:
Without change | withchange
---- | --------
![Screenshot from 2020-12-19 05-54-04](https://user-images.githubusercontent.com/71120226/102675228-9e475a00-41be-11eb-930e-415bc379385f.png) | ![Screenshot from 2020-12-19 05-55-26](https://user-images.githubusercontent.com/71120226/102675283-cf278f00-41be-11eb-8933-95c1aa93b670.png)


due to styling for mobile view which is not good without adding this line of code

Reply:
With change, there is no padding on Monday, 4th January

Reply:
U you say i can revert this change, but line below Monday does not looks gud

Reply:
There are other ways to remove the gap without removing all padding and margin